src/global/styles/mixins/pearl-chain-bullet.scss

```diff
@@ -27,61 +24,39 @@
 }
 
 @mixin pearl-chain-bullet {
-  height: var(--sbb-pearl-chain-bullet-size);
-  width: var(--sbb-pearl-chain-bullet-size);
-  background: var(--sbb-pearl-chain-bullet-color);
-  color: var(--sbb-pearl-chain-bullet-color);
-  border-color: var(--sbb-pearl-chain-bullet-color);
+  background: var(--sbb-pearl-chain-bullet-color-default);
+  color: var(--sbb-pearl-chain-bullet-color-default);
+  border-color: var(--sbb-pearl-chain-bullet-color-default);
   border-radius: 100%;
 }
 
-@mixin pearl-chain-bullet--xs {
-  --sbb-pearl-chain-bullet-size: var(--sbb-pearl-chain-bullet-size-xs);
-}
-
-@mixin pearl-chain-bullet--s {
-  --sbb-pearl-chain-bullet-size: var(--sbb-pearl-chain-bullet-size-s);
-}
-
-@mixin pearl-chain-bullet--m {
-  --sbb-pearl-chain-bullet-size: var(--sbb-pearl-chain-bullet-size-m);
-}
-
-@mixin pearl-chain-bullet--l {
-  --sbb-pearl-chain-bullet-size: var(--sbb-pearl-chain-bullet-size-l);
-}
-
-@mixin pearl-chain-bullet--xl {
-  --sbb-pearl-chain-bullet-size: var(--sbb-pearl-chain-bullet-size-xl);
+@mixin pearl-chain-bullet-start-stop {
+  height: var(--sbb-pearl-chain-bullet-size-start-end);
+  width: var(--sbb-pearl-chain-bullet-size-start-end);
 }
 
-@mixin pearl-chain-bullet-incidence {
-  --sbb-pearl-chain-bullet-color: var(--sbb-pearl-chain-bullet-color-incidence);
+@mixin pearl-chain-bullet-stop {
+  height: var(--sbb-pearl-chain-bullet-size-stop);
+  width: var(--sbb-pearl-chain-bullet-size-stop);
+  border: var(--sbb-pearl-chain-bullet-border-width) solid currentcolor;
+  background: Canvas;
 }
 
 @mixin pearl-chain-bullet-past {
-  --sbb-pearl-chain-bullet-color: var(--sbb-pearl-chain-bullet-color-past);
+  --sbb-pearl-chain-bullet-color-default: var(--sbb-pearl-chain-bullet-color-past);
 }
 
-@mixin pearl-chain-bullet-walk {
-  --sbb-pearl-chain-bullet-color: var(--sbb-pearl-chain-bullet-color-walk);
-}
-
-@mixin pearl-chain-bullet-transfer {
-  @include pearl-chain-bullet;
-
-  background: Canvas !important;
-  border: var(--sbb-pearl-chain-bullet-border-width) solid var(--sbb-pearl-chain-bullet-color);
+@mixin pearl-chain-bullet-irrelevant {
+  --sbb-pearl-chain-bullet-color-default: var(--sbb-pearl-chain-bullet-color-irrelevant);
 }
 
-@mixin pearl-chain-bullet-thin {
-  @include pearl-chain-bullet-transfer;
-
-  --sbb-pearl-chain-bullet-border-width: var(--sbb-pearl-chain-bullet-border-width-thin);
+@mixin pearl-chain-bullet-disruption {
+  --sbb-pearl-chain-bullet-color-default: var(--sbb-pearl-chain-bullet-color-disruption);
```

Comment:
For me it would sound more logical if the var --sbb-pearl-chain-bullet-color-default would be called --sbb-pearl-chain-bullet-color, because it's reassigned. but I leave it up to you